src/ts/components/seekbar.ts

```diff
@@ -48,6 +50,60 @@ export interface SeekBarMarker {
   duration?: number;
 }
 
+enum SeekBarType {
+  Vod,
+  Live,
+  Volume
+}
+
+interface ArrowKeyControls {
+  left: () => void;
+  right: () => void;
+  up: () => void;
+  down: () => void;
+}
+
+const arrowKeyControls = (currentValue: number, set: (number: number) => any, range: {min: number, max: number}): ArrowKeyControls => {
```

Comment:
this line is too long. When extracting this can be a function then we could break the line.